resources/views/components/card/address.blade.php

```diff
@@ -0,0 +1,37 @@
+@props(['shipping' => false, 'billing' => false, 'dynamicType' => false, 'check' => false])
+
+<address-card
+    {{ $attributes->whereStartsWith('v-') }}
+    @if(!$dynamicType)
+        :shipping="{{ var_export($shipping) }}"
+        :billing="{{ var_export($billing) }}"
+    @endif
+    v-slot="{ address, billing, shipping, isEmpty }"
+>
+    <div {{ $attributes->whereDoesntStartWith('v-')->class('relative bg-white border px-8 py-6 rounded max-w-[400px]') }} v-if="!isEmpty">
```

Comment:
should rounded also be prefixed with ct-?, you probably have to add the rounded to the checkout-theme tailwind config

Reply:
Also curious why you need the max-w class, can you show me in person why you need this?